sig-community-experience/meeting-template.md

```diff
@@ -0,0 +1,31 @@
+# Name of the Group/Governance Entity
+
+## Meta Information
+### Links, Resources, etc
+
+## Date & Time | Title in Calendar
+
+### Pre-meeting area
+#### Headline
+#### Main Topics to be discussed
+#### People/Teams/Skills needed
+#### Preparation
```

Comment:
Should we also add a comment here to illustrate what this is about / what should it contain?

Reply:
Yes, I will do.